styles/Navbar.module.css

```diff
@@ -0,0 +1,73 @@
+
+.header{
+    border-bottom: 1px solid #E2E8F0;
+}
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 1.5rem;
+}
+
+.none {
+    display: none;
+}
+
+.bar {
+    display: block;
+    width: 35px;
+    height: 3px;
+    margin: 5px auto;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: #222222;
+}
+
+.navmenu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    list-style-type: none;
+}
+
+.navitem {
+    margin-left: 5rem;
+}
+
+.navlink{
+    font-size: 1.5rem;
+    font-weight: 400;
+    color: #000000;
+    bottom: 20px;
+    text-decoration: none;
+}
+.navlink::after{
+    content:"";
+    position: absolute;
+    background-color: #F26522;
```

Comment:
if this is the original navbar color, can we use the variable instead of hard coding the value